.github/workflows/healthchecks_solr_ci.yml

```diff
@@ -4,15 +4,15 @@ on:
   push:
     #branches: [ master ]
     paths:
-      - src/HealthChecks.SolR/**
+      - src/HealthChecks.Solr/**
       - test/HealthChecks.SolR.Tests/**
       - .github/workflows/healthchecks_solr_ci.yml
     tags-ignore:
       - preview-*
   pull_request:
     #branches: [ master ]
     paths:
-    - src/HealthChecks.SolR/**
+    - src/HealthChecks.Solr/**

```

Comment:
Why so ?

Reply:
Sorry, i don't understand your question.

Reply:
I had to change `R` with `r` to pass CI build. On Windows, it does not matter, the build is successful, but not on Linux. I just became interested, whether this error remained unnoticed. Or did you see this error when working with `netcore-6.0` branch?